src/Server.php

```diff
@@ -38,12 +48,16 @@
      *                           of server
      * @param string $port       TCP port number
      * @param string $flags      Optional flags
+     * @param int $options       Connection options
+     * @param int $retries       Retries number
      * @param array  $parameters Connection parameters
      */
     public function __construct(
         string $hostname,
         string $port = '993',
         string $flags = '/imap/ssl/validate-cert',
+        int $options = 0,
+        int $retries = 1,
         array $parameters = []
```

Comment:
Please move the new options at the end of the signature to maintain backward compatibility

Reply:
Done.